Lib/test/test_unicodedata.py

```diff
@@ -175,8 +176,7 @@ def test_normalize(self):
         self.assertRaises(TypeError, self.db.normalize)
         self.assertRaises(ValueError, self.db.normalize, 'unknown', 'xx')
         self.assertEqual(self.db.normalize('NFKC', ''), '')
-        # The rest can be found in test_normalization.py
-        # which requires an external file.
+        # See also comprehensive tests in NormalizationTest below.
```

Comment:
```suggestion
```
I agree with the removal of the previous comment since this has been condensed into one file, but I don't think the new comment is needed.

Reply:
> Not strictly necessary, but I think there's a possible confusion where you're looking through all these tests which test one function after another, and this fits right in, and so you might naturally infer that this is the test (or the main test) for this function, and not go looking for others.

Hmm, fair point. That could potentially be useful for readers to know.

I'm not completely against this comment remaining, but I think the other two (especially "Perform tests" on line 370) should be removed. Comments that don't provide additional information or context end up adding unneeded noise/clutter to the code. Each individual one isn't much of an issue, but it can definitely add up over time.

Reply:
I would suggest completing the unification by putting this case in `NormalizationCases` (and renaming it `test_edge_cases` or something; that's what it seems to be about).

Reply:
Sure -- done.